### DISCUSSION TITLE
New feature: ability to move tabs between windows

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#3380] [Announcements]
Thanks to efforts of @avorobey In latest [pre-release version](https://www.sumatrapdfreader.org/prerelease) you can:

* move tabs between window
* drag tabs within window
* drag out a tab to create a new window.

Also tweaked the display of tabs.for bigger contrast between tab and background, so it's easier to tell selected tab.

**Comment by linbdnil** [comment]
I love this feature. Current issue is it become easily crash (if not always) when using forward-search of latex (the inverse-search work well), my tool combination is vscode + latex workshop + miktex + sumatrapdf (build 15374), and the 3.4.6 version work well with both forward-search and inverse-search.

**Comment by venkat2305** [comment]
Please add dark mode too

**Comment by CalebSpencerTorch** [comment]
I tested the pre-release with this feature. It worked well until it crashed so I'm looking forward to further development of this feature.

**Reply by GitHubRulesOK** [reply]
@CalebSpencerTorch 
for a crash its best to follow the guide that pops up (on cancel)  and supply the details **if its repeatable** otherwise your cause may not be addressed, the more good **quality** repeatable crash submittals the better the quality of next release
